draft-ietf-httpbis-origin-frame.md

```diff
@@ -58,6 +58,11 @@ Using a status code in this manner allows clients to recover from misdirected re
 penalty of adding latency. To address that, this specification defines a new HTTP/2 frame type,
 "ORIGIN", to allow servers to indicate what origins a connection is usable for.
 
+Additionally, experience has shown that HTTP/2's requirement to establish server authority using
```

Comment:
in discussions on this there are a couple of other motivators that may (or may not!) be worth including as introductory text.. one is simply that it can save a dns lookup by effectively inlining the result and the other is that dns is often a privacy leak that can be avoided by not making the lookup